src/main/resources/templates/product/productDetails.html

```diff
@@ -154,8 +214,8 @@ <h4 class="mdl-dialog__title">Archive this product?</h4>
 </dialog>
 <!--logic for showing popup-->
 <script>
-    var dialog = document.querySelector('dialog');
-    var showModalButton = document.querySelector('.show-modal');
+    let dialog = document.querySelector('dialog');
```

Comment:
you're right in changing the var. But there's still only one popup, so should be 'const'

Reply:
fixed